goldendict.vcxproj.filters

```diff
@@ -62,9 +62,6 @@
     <ClCompile Include="article_netmgr.cc">
       <Filter>Sources</Filter>
     </ClCompile>
-    <ClCompile Include="articleinspector.cc">
-      <Filter>Sources</Filter>
-    </ClCompile>
```

Comment:
#1450 removes goldendict.sln, goldendict.vcxproj and goldendict.vcxproj.filters claiming that they are not maintained and not useful. Do the changes in this PR make these files up to date and working? If not, I suppose these files should not be touched at all in this PR.

Reply:
the code used in articleinspector.cc is webengine related .it does not work in webengine.